### PR TITLE
Detail page layout

### PR DESCRIPTION
**Pull request by ellenkramp**
Could I get initial 👀 on this layout? @rfpoulos and @ponchieponcho ?

### CODE REVIEW

**Comment by ellenkramp** [comment]
@rfpoulos @ponchieponcho

File: css/detail.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+#map {
+    background: yellow;
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    margin-top: 1%;
+    align-items: center;
+}
```
Comment (rfpoulos) [inline css/detail.css]: must have height

File: javascript/detail.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+var fireBaseCityRef = firebase.database().ref('routes/' + objectId + '/city');
+var fireBaseStateRef = firebase.database().ref('routes/' + objectId + '/state');
+var fireBaseTitleRef = firebase.database().ref('routes/' + objectId + '/title');
+var fireBaseUserIdRef = firebase.database().ref('routes/' + objectId + '/userID');
+var fireBaseThumbnailRef = firebase.database().ref('routes/' + objectId + '/thumbnail');
+
+var docTitle = document.getElementById('title');
+fireBaseTitleRef.on('value', function(snapshot) {
+    docTitle.textContent = snapshot.val();
+});
```
Comment (rfpoulos) [inline javascript/detail.js]: var docTitle = document.getElementById('title');
docTitle.textContent = fireBaseTitleRef.val;

?

File: javascript/detail.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+var docUserId = document.getElementById('userId');
+fireBaseUserIdRef.on('value', function(snapshot) {
+    docUserId.textContent = snapshot.val();
+});
+
+var userImage = document.getElementById('user-image');
+fireBaseThumbnailRef.on('value', function(snapshot) {
+    userImage.src = snapshot.val();
```
Comment (rfpoulos) [inline javascript/detail.js]: better not to set image SRC better to set attribute.  don't have to set DOM directly.

**Comment by rfpoulos** [comment]
looks good!  go ahead an merge

**Pull request merged**